src/session/session_api.c

```diff
@@ -291,7 +291,13 @@ __session_reconfigure(WT_SESSION *wt_session, const char *config)
 	WT_SESSION_IMPL *session;
 
 	session = (WT_SESSION_IMPL *)wt_session;
-	SESSION_API_CALL(session, reconfigure, config, cfg);
+	/*
+	 * Even though this operation is not allowed in prepared transaction
+	 * state, true is passed indicating the operation is allowed in
```

Comment:
I suggest removing some of the repetitiveness in this comment:
* Even though this operation is not allowed while the transaction is prepared,
* we pass in true so that the running transaction check below takes precedence
* over the prepared transaction check.